File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.java
// Last updated: 5/20/2026, 11:36:09 PM


import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

class Solution {
    public int maximumWealth(int[][] accounts) {
        List<Integer> wealth = new ArrayList<>();
        for (int[] account : accounts) {
            int sum = 0;
            for (int i : account) {
                sum += i;
            }
            wealth.add(sum);
        }
        return Collections.max(wealth);
    }
}

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.cpp
// Last updated: 5/20/2026, 11:36:06 PM
class Solution {
public:
  int maximumWealth(vector<vector<int>> &accounts) {
    int max{};
    for (const auto &account : accounts) {
      int sum = std::accumulate(account.begin(), account.end(), 0,
                                [](int acc, int i) { return acc + i; });
      max = std::max(max, sum);
    }

    return max;
  }
};
